src/core/lib/gpr/log.cc

```diff
@@ -109,6 +110,8 @@ static gpr_atm parse_log_severity(absl::string_view str, gpr_atm error_value) {
 }
 
 void gpr_log_verbosity_init() {
+  absl::InitializeLog();
```

Comment:
`grpc_init()` calls this. I thought this would be soon enough, but it doesn't look like it is.

Reply:
Can you try doing this in [Init_grpc_c](https://github.com/grpc/grpc/blob/master/src/ruby/ext/grpc/rb_grpc.c#L306)?

Reply:
Yes, that seems to work, though calling `absl::InitializeLog()` twice causes the library to fail the seoncd time.